frontend/packages/policy-editor/src/components/PolicyCardRules/PolicyRule/ExpandablePolicyElement/ExpandablePolicyElement.tsx

```diff
@@ -68,7 +72,14 @@ export const ExpandablePolicyElement = ({
           onFocus={() => setIsButtonFocused(true)}
           onBlur={() => setIsButtonFocused(false)}
         >
-          <StudioLabelAsParagraph size='small'>{title}</StudioLabelAsParagraph>
+          <div className={classes.headerWrapper}>
+            <StudioLabelAsParagraph size='small'>{title}</StudioLabelAsParagraph>
```

Comment:
I believe Designsystemet has changed `small` to `sm` now 😄 
```suggestion
            <StudioLabelAsParagraph size='sm'>{title}</StudioLabelAsParagraph>
```